src/server/api/conversations.js

```diff
@@ -3,6 +3,7 @@ import { config } from "../../config";
 import { Assignment, r } from "../models";
 import { addWhereClauseForContactsFilterMessageStatusIrrespectiveOfPastDue } from "./assignment";
 import { buildCampaignQuery } from "./campaign";
+import { UNASSIGNED_TEXTER } from "../../components/IncomingMessageFilter";
```

Comment:
I think, given the previous things build issues we've dealt with, it's best not to do server side imports from jsx. I could be persuaded if there were substantial benefit, but it's easy enough to just reproduce this constant either in this file or on a server side constants file

Reply:
That's fine! I think defining it in this file is fine too, I don't think there's benefits to extracting it if the constants aren't shared across multiple files

Reply:
Oh sure yeah!